charts/victoria-metrics-k8s-stack/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 ## Next release
 
-- TODO
+- Added alertmanager datasource. See [this issue](https://github.com/VictoriaMetrics/helm-charts/issues/1592)
+- Renamed `grafana.sidecar.dashboards.additionalDashboardLabels` to `grafana.sidecar.dashboards.labels`
+- Renamed `grafana.sidecar.dashboards.additionalDashboardAnnotations` to `grafana.sidecar.dashboards.annotations`
+- Renamed `grafana.sidecar.datasources.default` to `grafana.sidecar.datasources.victoriametrics`
+- Renamed `grafana.additionalDataSources` to `grafana.sidecar.datasources.extra`
```

Comment:
I prefer not to change this, the original value name looks ok to me. I remember some users using the additionalDataSource as the default one and this change could breaks their setup.

Reply:
moved this value to defaultDatasources as it's not a part of grafana chart as well